code/modules/vehicles/motorbike.dm

```diff
@@ -195,8 +197,7 @@
 /obj/vehicle/ridden/motorbike/projectile_hit(obj/projectile/P)
```

Comment:
could you please move that proc to projectile.dm ? 

It's where projectile_hit are supposed to be.

Reply:
Moved.